### ISSUE TITLE
Incorrect Assertion in TensorFlow Lab1

### ISSUE THREAD

**Issue by autojazari** [udacity/sdc-issue-reports#12]
The assertions in
```
assert features._shape == (None) or features._shape == (None, 784), 'The of features is incorrect'
assert labels._shape == (None) or features._shape == (10), 'The shape of labels is incorrect
```
`TensorFlow lab1` fail when the shapes are correct

```
features = tf.placeholder(tf.float32, shape=(None, 784))
labels = tf.placeholder(tf.float32, shape=(None))
print(features._shape)
 (?, 784)
```
Using `TensorFlow` version [tensorflow-0.11.0rc1](https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0rc1-cp35-cp35m-linux_x86_64.whl) CPU only

@dalex01 Note: commenting out these two assertions I am able to create `Accuracy Function`, and begin changing `learning_rate`, etc...

**Comment by Brok-Bucholtz** [comment]
@autojazari 

Thanks for the feedback.  This is fixed in the newest version.

**Comment by autojazari** [comment]
@Brok-Bucholtz My pleasure! If you have a moment, please checkout the plot for [my solution](https://github.com/autojazari/sdc-lab1-notmnist) of this assignment and give quick feedback

**Comment by Brok-Bucholtz** [comment]
@autojazari 
Nice graph!  50 epochs and ~180 batch size, how long did that take?  If you want to make more graphs, check out [TensorBoard](https://www.tensorflow.org/how_tos/summaries_and_tensorboard/index.html).  I would've used it over matplotlib, but it doesn't support jupyter.

**Comment by autojazari** [comment]
@Brok-Bucholtz :+1: on TensorBoard, thanks!

The batch size was closer to 1800, not sure if I missed something or that was a typo.  I set it to almost 1%.  On Dell XPS with i-7 was only 95 seconds.

I did set it at first to .01% for testing and it took quite a few minutes.  The loss function looked like an EKG :)
